### DISCUSSION TITLE
Modeling with Percent Changes (positive and negative values)

### DISCUSSION THREAD

**Discussion by abadgerw** [easystats/parameters#760] [Q&A]
I hope this message finds the easystats community well. Thank you for your lovely resource. I am interested in two things:

1. Modeling percent change (in this case volume changes between two time points) as my dependent variable to understand what factors predict this. 

2. Using percent change (in this volume changes between two time points) as an independent variable to understand whether it predicts future disability. 

Is there a model type that is most appropriate for case #1 besides a regular linear model? I was looking at the following guidance (http://htmlpreview.github.io/?https://github.com/strengejacke/mixed-models-snippets/blob/master/overview_modelling_packages.html) but couldn't find the best approach for dependent variables that represented percent changes which could take on positive and negative values. 

For case #2, are there any caveats to using a percent change value that can be positive or negative as an independent variable in a model?

**Comment by strengejacke** [comment]
I would model the change between timepoints by adding a `time` predictor. Your outcome seems to be a proportion, so it sound like _beta-regression_ is something you should consider. I personally would model "growth" models (or multiple measurements) as mixed model:

`outcome ~ time + <other predictors> + (1 + time | subject_ID)`

See also https://rpsychologist.com/r-guide-longitudinal-lme-lmer. In your case, packages _glmmTMB_ or _brms_ look appropriate here.

Regarding 2) What is your research question / model design (outcome etc.)?

**Comment by bwiernik** [comment]
If you have the raw values for the response, then a lognormal model or log-transformed normal model will capture percent changes in the response per 1-unit change in the predictors. 

If you only have the percentages, then like Daniel suggests, a beta family regression would be most appropriate, but this has a downside that you lose information about how many observations contribute to each percent. 50% of 1000 observations is much more reliable an estimate than 50% of 10 observations.

**Reply by strengejacke** [reply]
True, percentage change may also refer to a continuous outcome that is not a proportion.

**Comment by abadgerw** [comment]
Thank you both! My outcome is a percent change from baseline to follow-up which can be positive or negative. Can a beta model handle both positive a negative values? One way around this is that the positive values don't exactly have biological meaning and basically just mean 0 (ie no atrophy) while how big negative values are indicate an amount of atrophy. Therefore, I could think of a zero-inflated beta?

I thought about a mixed model but this MRI algorithm derives a percent change directly from two registered MRI images rather than tabulating the difference between the cross-sectional areas of two images. Therefore, I only have one value. Therefore, I think I would need to use regular regression?

**Reply by bwiernik** [reply]
Okay, in that case, you effectively are dealing with a model with a log(follow-up) response and a log(baseline) offset. A normal linear regression model would be fine then (a lognormal family seems reasonable to me for brain volume).
